(#770) Clarify Package Environment variables and their contents (#764)
chocolatey/docs: input/en-us/guides/upgrading-to-chocolatey-v2-v6.md

### Package Version Normalization

> :choco-note: **NOTE**
>
> **Important information for package maintainers.** The `ChocolateyPackageVersion` environment variable follows the below version normalization. This means if your package scripts expect version numbers that fall outside of the specified format, then they may not behave as expected.

> 

Reviewer: Can we link to the variable on the functions page, here?

Author: I've linked to the environment variables section, and reworded this a bit as it felt a little odd.

> > **Important information for package maintainers.** The `ChocolateyPackageVersion` [environment variable](xref:powershell-reference#environment-variables) follows the below version normalization. If your package scripts use this variable, and the nuspec specified version falls outside of the specified format, then they may not behave as expected. It is recommended to use specific version numbers within your scripts and not depend on these environment variables.

Reviewer 2: ```suggestion
> **Important information for package maintainers.** The `ChocolateyPackageVersion` [environment variable](xref:powershell-reference#environment-variables) follows the below version normalization. If your package scripts use this variable, and the nuspec specified version falls outside of the normalized format, then they may not behave as expected. We recommend using specific version numbers within your scripts rather than depending on these environment variables.
```

Author: I don't know how I didn't get this updated Friday, but it's updated now 😁
